sources/models/shared-defaults/oci-defaults-docker-capabilities.toml

```diff
@@ -0,0 +1,17 @@
+[settings.oci-defaults.capabilities]
+# These values represent the default capabilities in the default 
+# OCI spec for docker.
+audit-write = true
+chown = true
+dac-override = true
+fowner = true
+fsetid = true
+kill = true
+mknod = true
+net-bind-service = true
+net-raw = true
+setgid = true
+setfcap = true
+setpcap = true
+setuid = true
+sys-chroot = true
```

Comment:
If this is identical to `oci-defaults-containerd-cri-capabilities.toml`, we should just rename that one and have one source of truth.